app/i18n/it/conf.php

```diff
@@ -105,7 +105,13 @@
 	),
 	'profile' => array(
 		'_' => 'Gestione profili',
-		'api' => 'Gestione API',
+		'api' => array(
+			'_' => 'Gestione API',
+			'check_link' => 'Verificare lo stato dell'API tramite: <kbd><a href="../api" target="_blank">%s</a></kbd>',
+			'disabled' => 'L'accesso all'API è disabilitato.',
+			'documentation_link' => 'See the <a href="https://freshrss.github.io/FreshRSS/en/users/06_Mobile_access.html#access-via-mobile-app" target="_blank">documentation and list of known apps</a>',	// TODO
```

Comment:
```suggestion
			'documentation_link' => 'Vedi <a href="https://freshrss.github.io/FreshRSS/en/users/06_Mobile_access.html#access-via-mobile-app" target="_blank">l’elenco delle applicazioni</a>',
```
In the translation I changed the meaning of the text slightly
If it doesn't fit I'll translate it differently

English translation of the phrase I changed: "See the list of applications"

Reply:
The idea here is that the link has the documentation AND has the link to the apps list

Reply:
```suggestion
			'documentation_link' => 'Vedi la <a href="https://freshrss.github.io/FreshRSS/en/users/06_Mobile_access.html#access-via-mobile-app" target="_blank">documentazione e l'elenco delle applicazioni</a>',	// DIRTY
```

This is a translation more similar to the original

If you give me more details about how and where you will use this phrase I can be more precise with the translation

Reply:
@UserRoot-Luca Please remember to use the *typographic apostrophe* `’` and not the *vertical apostrophe* `'`

Reply:
```suggestion
			'documentation_link' => 'Vedi la <a href="https://freshrss.github.io/FreshRSS/en/users/06_Mobile_access.html#access-via-mobile-app" target="_blank">documentazione e l’elenco delle applicazioni</a>',	// DIRTY
```
@Alkarex Sorry is I forgot about it I'll solve it right away

Reply:
When you have updated the translation, you may remove the `// DIRTY` at the end of the line

Reply:
```suggestion
			'documentation_link' => 'Vedi la <a href="https://freshrss.github.io/FreshRSS/en/users/06_Mobile_access.html#access-via-mobile-app" target="_blank">documentazione e l’elenco delle applicazioni</a>',
```